.circleci/config.yml

```
@@ -0,0 +1,62 @@
+version: 2.1
+executors:
+  infrastructure_container:
+    docker:
+      - image: aeternity/infrastructure:master
```

Why :master? See also https://github.com/aeternity/terraform-aws-aenode-deploy/pull/1/files#diff-1d37e48f9ceff6d8030570cd36286a61R5
```suggestion
      - image: aeternity/infrastructure
```